macros/models/_sector/nft/platforms/element_v1_base_trades.sql

```diff
@@ -0,0 +1,123 @@
+-- Element NFT trades (re-usable macro for all chains)
+{% macro element_v1_base_trades(erc721_sell_order_filled, erc721_buy_order_filled, erc1155_sell_order_filled, erc1155_buy_order_filled) %}
+
+
+SELECT
+  DATE_TRUNC('day', evt_block_time) AS block_date,
```

Comment:
should we apply [these](https://github.com/duneanalytics/spellbook/pull/3705#issuecomment-1629061317) changes in this PR? otherwise, i'm not sure partitioning will work?